### ISSUE TITLE
Token Auth for Azure SQL Server to Store ASP.NET Session State

### ISSUE THREAD

**Issue by c767695** [aspnet/AspNetCore#10838]
Is there a feature in the roadmap already for in the roadmap to implement a layer to allow developers to use Azure SQL token authorization for the Sql Provider to store the ASP.NET Session State? Are there other providers for Azure that fill a similar role such as Azure Cache for Redis.

**Comment by davidfowl** [comment]
Is this for ASP.NET Core? or ASP.NET?

**Comment by c767695** [comment]
This is in ASP.NET. Does this need to thus move to another repo?

**Comment by davidfowl** [comment]
Yes. https://github.com/aspnet/AspNetSessionState, we should be able to transfer the issue
